src/test/scala/io/smartdatalake/config/TestFileTransformer.scala

```diff
@@ -25,11 +25,12 @@ class TestFileTransformer extends CustomFileTransformer {
   /**
    * This method has to be implemented by the custom transformer to define the transformation.
    *
+   * @paran options
```

Comment:
```suggestion
   * @param options
```

Reply:
Didn't fully understand what kind of options you can provide either. Maybe this is the correct place to add some comments about that?